Pipfile

```diff
@@ -6,23 +6,21 @@ name = "pypi"
 [packages]
 docker = "*"
 python-gitlab = "*"
-requests-oauthlib = "*"
 flask = "*"
 escapism = "*"
 kubernetes = "*"
 gunicorn = "*"
 gevent = "*"
 requests = "*"
 pyyaml = ">=4.2b1"
-raven = {extras = ["flask"], version = "*"}
-blinker = "*"
 jupyterhub = "==0.9.6"
 "urllib3" = ">=1.24.2"
+sentry-sdk = {extras = ["flask"],version = "*"}
 
 [dev-packages]
-chartpress = "*"
 pylint = "*"
 yapf = "*"
+chartpress = {editable = true,git = "https://github.com/jupyterhub/chartpress",ref = "a81ce038f7b22d2cc99ff0d25a28b24b92bb7dbe"}
```

Comment:
I remember that Fotis had a fork of `chartpress`. Should we use our fork here?

Reply:
all the changes that we've made to chartpress have been merged to master - that's what is being used here.